v2/openapi/fb-provider-collateral-api.yaml

```diff
@@ -720,18 +820,38 @@ components:
         - **Credited** - The transaction has been completed successfully and the account has been credited
         
         - **Rejected** - The transaction has been rejected and the account has not been credited
+    CollateralDepositTransactionIntentRequest:
+      type: object
+      required: [ asset, amount, approvalRequest ]
+      properties:
+        asset:
+          $ref: '#/x-schemas/CryptocurrencyReference'
+        amount:
+          $ref: '#/x-schemas/PositiveAmount'
+        approvalRequest:
+              $ref: '#/components/schemas/ApprovalRequest'
+    CollateralDepositTransactionIntentResponse:
+      allOf:
+        - type: object
+          required: [id, status]
+          properties:
+            id:
+              type: string
+            status:
+              $ref: '#/components/schemas/CollateralTransactionIntentStatus'
+        - $ref: '#/components/schemas/CollateralDepositTransactionIntentRequest'
```

Comment:
We said we're currently not going to have an intent response per endpoint family